textmetric/chrFpp.py

```diff
@@ -0,0 +1,175 @@
+import sys
```

Comment:
The code is adopted from Popovic with very slight modifications, right?

Reply:
The most of code is copied from that source, yes. The modifications from last commit are provided by @AntonPrazdnichnykh, be waiting for his answer.

Reply:
@mishaevtikhiev, yes, it's right. Specify your question, please, if you expect a more detailed answer.